docs/.vuepress/config.ts

```diff
@@ -25,6 +25,12 @@ export default defineUserConfig<DefaultThemeOptions> ({
                 '/recipes/getting-started.md'
               ]
             },
+            {
+              text: 'Ingredients',
+              children: [
+                '/recipes/ingredients/get-program-accounts.md'
```

Comment:
We should probably make the url under `/ingredients` instead of `/recipes`. I believe we're going to move the getting started stuff as well